examples/freetype_gl/single_storage/src/texture_manager.ads

```diff
@@ -0,0 +1,18 @@
+
+with GL.Buffers;
+with GL.Objects.Buffers;
+with GL.Objects.Textures;
+with GL.Types;
+
+package Texture_Manager is
+
+   subtype V_Buffer is GL.Objects.Buffers.Buffer;
+   --  2D quad as two triangles requires 2 * 3 vertices of 4 floats
+   subtype Vertex_Array is GL.Types.Singles.Vector4_Array (1 .. 6);
+
+   procedure Setup_Graphic (Vertex_Buffer : in out V_Buffer;
+                            aTexture      : in out GL.Objects.Textures.Texture;
+                            X, Y          : GL.Types.Single;
+                            Scale         : GL.Types.Single;
+                            Char          : Character := 'g');
+end Texture_Manager;
```

Comment:
Is this shareable with your previous PR? could be in `examples/freetype_gl/common`.

Reply:
I'll look into this.

Reply:
Something seems to have gone drastically wrong.
This code seems to be just a copy of the previous PR.
I'll have to see if I can recover the real code tomorrow.
If I can't fix it, I'll have to withdraw this PR and move on to the string example.

Reply:
I think I've fixed it, thanks to git revert.